hdf5-types/Cargo.toml

```diff
@@ -8,13 +8,18 @@ description = "Native Rust equivalents of HDF5 types."
 repository = "https://github.com/aldanor/hdf5-rust"
 homepage = "https://github.com/aldanor/hdf5-rust"
 edition = "2018"
+build = "build.rs"
 
 [features]
-const_generics = []
+const-generics = []
+force-h5-allocator = []
 
 [dependencies]
 ascii = "1.0"
```

Comment:
As for various 3rd party scripts - try running `cargo outdated -R -w` to see if some versions can/need to be bumped? (since we're already changing a lot)